content/en/docs/usage/certificate.md

```diff
@@ -35,6 +35,20 @@ metadata:
 spec:
   # Secret names are always required.
   secretName: example-com-tls
+
+  # Secret template is optional. If set, these annotations
+  # and labels will be copied to the secret named example-com-tls.
+
+  # Note: Once created, labels and annotations are not yet removed
```

Comment:
Added this note to keep clear we currently don't remove annotations from the secret if secretTemplate changes.

Reply:
Is it also necessary to point out that if added or changed they are only updated when the cert is renewed?

Reply:
Thanks, @benlangfeld, updated as suggested.